src/server/services/import/import.js

```diff
@@ -29,6 +28,35 @@ const importData = async (dataRaw, { slug, format, user, idField }) => {
   let data = await parseInputData(format, dataRaw, { slug });
   data = toArray(data);
 
+  let failures = [];
+  let importFunction;
+  if (slug === 'api::candidate.candidate') {
+    importFunction = async (datum) => await createOrUpdateCandidate(datum, { slug });
+  } else if (slug === 'api::nomination.nomination') {
+    importFunction = async (datum) => await createOrUpdateNomination(datum, { slug });
+  } else {
+    failures.push('Slug not supported');
```

Comment:
I guess this will make the call to the function throw later on, so maybe throw the error already here?